src/midi/midi_fluidsynth.cpp

```diff
@@ -491,6 +610,34 @@ static void setup_reverb(fluid_synth_t* synth)
 
 MidiDeviceFluidSynth::MidiDeviceFluidSynth()
 {
+	DynLibResult res = load_dynlib_fluid();
+	switch (res) {
+		case DynLibResult::Success:
+			break;
+		case DynLibResult::LibOpenErr: {
+			const auto sym_msg = "FSYNTH: Failed to load fluidsynth library";
+			LOG_ERR("%s", sym_msg);
+			throw std::runtime_error(sym_msg);
+			break;
+		}
+		case DynLibResult::ResolveSymErr: {
+			const auto sym_msg = "FSYNTH: Failed to resolve one or more fluidsynth symbols";
+			LOG_ERR("%s", sym_msg);
+			throw std::runtime_error(sym_msg);
+			break;
+		}
+	}
+
+	int v_maj, v_min, v_micro;
```

Comment:
minor: maybe spell out "major" and "minor" too

Reply:
I'm now using a version struct.